harvest/trader/trader.py

```diff
@@ -406,6 +403,56 @@ def _update_stats(self, df_dict, new=False, option_update=False):
             equity = net_value + self.account['cash']
             self.account['equity'] = equity
 
+    def storage_setup(self, interval):
```

Comment:
not sure why these were removed,maybe a problem when i was fixing merge conflicts, but used to setup the storage.

Reply:
ooooh that explains a lot, I was wondering why the storage was not being initialized at first. If you want you can remove this or update it since I already added a (in my opinion) a more minimal implementation of this code in an earlier PR: https://github.com/tfukaza/harvest/blob/462fcee387fa924709372545b2554a1795f913e6/harvest/trader/trader.py#L117-L120

Reply:
Got it, I'll remove it.